src/api_callable.ts

```diff
@@ -121,9 +128,14 @@ export class Canceller {
   }
 }
 
+// tslint:disable-next-line no-any
+export interface CancellablePromise<T = any> extends Promise<T> {
```

Comment:
Maybe remove the `= any` since this is a new type? I don't think we should allow it to implicitly be any if we have control over it.